server/domain/routers/wind.go

```diff
@@ -0,0 +1,150 @@
+package routers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"school-project-2019/server/domain/devices"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func WindInit(router *httprouter.Router) {
+	router.GET("/wind", GetWindSensor)
+	router.GET("/wind/events", FindWindEvents)
+	router.POST("/wind/event", CreateWindEvent)
+	router.GET("/wind/event/last", GetLastDate)
+	router.PUT("/wind", UpdateWindSensor)
+}
+
+func GetWindSensor(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	wind := devices.Wind{}
+	device, err := wind.Get()
+	// testing custom error response
+	if err == devices.NOT_FOUND {
+		http.Error(w, errors.New("the device is not found").Error(), http.StatusNotFound)
+		return
+	}
+
+	response, err := json.Marshal(device)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Write(response)
+}
+
+func UpdateWindSensor(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	defer r.Body.Close()
+
+	wind := devices.Wind{}
+	_, err := wind.Get()
+	if err == devices.NOT_FOUND {
+		http.Error(w, errors.New("the device is not found").Error(), http.StatusNotFound)
```

Comment:
Just add a plain text here, no need to make an error and then get the text from this error